templates/workflow/version.tpl

```diff
@@ -0,0 +1,54 @@
+{**
+ * lib/pkp/templates/workflow/version.tpl
+ *
+ * Copyright (c) 2014-2016 Simon Fraser University Library
+ * Copyright (c) 2003-2016 John Willinsky
```

Comment:
The copyright year and name should be checked everywhere -- I didn't mentioned it everywhere...